Source/Core/InputCommon/ControllerEmu/Setting/NumericSetting.h

```diff
@@ -3,7 +3,10 @@
 
 #pragma once
 
+#include <algorithm>
 #include <atomic>
+#include <functional>
+#include <mutex>
 #include <string>
```

Comment:
```suggestion
#include <string>
#include <utility>
#include <vector>
```

(For `std::pair` and `std::vector`)